app/v1/models/request/NinoApplication.scala

```diff
@@ -19,30 +19,32 @@ package v1.models.request
 import play.api.libs.functional.syntax._
 import play.api.libs.json._
 
-case class NinoApplication(
-                            nino: String,
-                            gender: Gender,
-                            entryDate: DateModel,
-                            birthDate: DateModel,
-                            birthDateVerification: BirthDateVerification,
-                            officeNumber: String,
-                            contactNumber: Option[String],
-                            country: Int,
-                            name: NameModel,
-                            historicNames: Option[Seq[NameModel]],
-                            address: AddressModel,
-                            historicAddresses: Option[Seq[AddressModel]],
-                            marriages: Option[Seq[Marriage]],
-                            originData: Option[OriginData],
-                            priorResidency: Option[Seq[PriorResidencyModel]],
-                            abroadLiability: Option[AbroadLiabilityModel]
+case class NinoApplication(nino: String,
+                           gender: Gender,
+                           entryDate: DateModel,
+                           birthDate: DateModel,
+                           birthDateVerification: Option[BirthDateVerification],
+                           officeNumber: String,
+                           contactNumber: Option[String],
+                           country: Int,
```

Comment:
Looking at the schema this field has now been removed and looks to be replaced with the new nationality code which you've already added. I reckon it's probably worth removing this within this task.

Reply:
Done